rocksdb_admin/helix_client.cpp

```diff
@@ -27,17 +27,18 @@
 #include "common/network_util.h"
 
 DECLARE_int32(port);
+DEFINE_string(helix_log_config, "", "Participant log config");
```

Comment:
essentially this can be any flags passed to JVM? If yes, shall we rename it?

Reply:
updated